packages/build/package.json

```diff
@@ -7,9 +7,8 @@
     "netlify-build": "./src/core/bin.js"
   },
   "files": [
-    "src",
-    "!*.test.*",
-    "!*~"
+    "src/**/*.js",
+    "src/**/*.yml"
```

Comment:
Apparently, `eslint-plugin-node` does not support using `"src/**/*.{js,yml}"` (even though `npm` does).